### PR TITLE
solved 1512, 2011, 2114

### PR DESCRIPTION
**Pull request by yurihoka**
Description:
1512 - https://leetcode.com/problems/number-of-good-pairs/description/
2011 - https://leetcode.com/problems/final-value-of-variable-after-performing-operations/description/
2114 - https://leetcode.com/problems/maximum-number-of-words-found-in-sentences/description/

### CODE REVIEW

**Comment by yurihoka** [comment]
please do not care about no.66

File: 1512.js
Comment (HidemichiShimura) [inline 1512.js:1]: Well done 👍

File: 2011.js
Comment (HidemichiShimura) [inline 2011.js:1]: Well done 👍

File: 2114.js
Comment (HidemichiShimura) [inline 2114.js:1]: Well done 👍

**Review by HidemichiShimura — APPROVED** [review]

**Pull request merged**